{
  "project": {
    "name": "libtiff"
  },
  "name": "CVE-2017-7600",
  "binary": "/data/vulnloc/libtiff/CVE-2017-7600/src/tools/tiffcp",
  "crash": {
    "command": "-i $POC out.tiff",
    "input": "/CrashRepair/experiments/vulnloc/libtiff/CVE-2017-7600/tests/1.tif",
    "extra-klee-flags": "--link-llvm-lib=/CrashRepair/lib/libjpeg-8.4.bca",
    "expected-exit-code": 1,
    "bad_output":"runtime error: -115 is outside the range of representable values"
  },
  "source-directory": "src",
  "build": {
    "directory": "src",
    "binary": "/data/vulnloc/libtiff/CVE-2017-7600/src/tools/tiffcp",
    "sanitizerflags": "",
    "commands": {
      "prebuild": "exit 0",
      "clean": "make clean",
      "build": "make CFLAGS=\"-fsanitize=float-cast-overflow ${INJECT_CFLAGS}\" CXXFLAGS=\"-fsanitize=float-cast-overflow ${INJECT_CXXFLAGS}\" LDFLAGS=\"${INJECT_LDFLAGS}\""
    }
  },
   "fuzzer": {
    "seed": 3,
    "crash-tag": "runtime;outside the range of representable values",
    "mutate-range": "default",
    "timeout": {
      "local": 300,
      "global": 300
    },
    "proof-of-crash": {
      "format": ["bfile"],
      "values": ["/CrashRepair/experiments/vulnloc/libtiff/CVE-2017-7600/tests/1.tif"],
      "commands": {
        "crash": ["/data/vulnloc/libtiff/CVE-2017-7600/src/tools/tiffcp", "-i", "***", "out1.tiff"],
        "trace": ["/data/vulnloc/libtiff/CVE-2017-7600/src/tools/tiffcp", "-i", "***", "out2.tiff"]
      }
    }
  }
}
